resources/views/layouts/app.blade.php

```diff
@@ -23,6 +23,12 @@
               <a class="nav-link" href="{{ route('teams.index') }}">Teams</a>
             </li>
 
+            @can('access-nova')
+            <li class="nav-item">
+              <a class="nav-link" href="/nova/">Nova</a>
```

Comment:
Are we calling this Nova to normal people?

Reply:
It already says Nova a few places in the UI.

I'm open to other suggestions